.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  # Intentionally not adding automatic NPM updates because I do
+  # not trust the plugin's tests to catch issues caused by updates.
+  - package-ecosystem: "maven"
+    directory: "/"
+    reviewers:
+      - "dwnusbaum"
+    schedule:
+      interval: "daily"
```

Comment:
maybe only weekly?

Reply:
I think that most of the dependencies here have their version controlled via https://github.com/jenkinsci/bom, so we shouldn't see too many updates. Maybe we can try daily for now and move to weekly if there is too much noise?

Reply:
yup go for it. I just sometimes receive too many dependabot email. I wonder if gmail will flag this as a spam :)